src/app/types/Authorization.ts

```diff
@@ -164,6 +164,13 @@ export interface SiteAuthorizationActions extends AuthorizationActions {
   canUnassignUsers?: boolean;
 }
 
+export interface SiteAreaAuthorizationActions extends AuthorizationActions {
+  canAssignChargingStations?: boolean;
+  canUnassignChargingStations?: boolean;
+  canAssignAssets?: boolean;
+  canUnassignAssets?: boolean;
+}
+
```

Comment:
This is where `exportOCPPParamsAction` and `siteAreaGenerateQrCodeConnectorAction` should also go in the future?

Reply:
Well, maybe it'd make more sense on `AuthorizationActions` since it's a common action.